jax/interpreters/pxla.py

```diff
@@ -1832,6 +1832,49 @@ def _pmap_lowering(ctx, *in_nodes, axis_name,
 # ------------------- xmap -------------------
 
 class Mesh(ContextDecorator):
+  """Declare the hardware resources available in the scope of this manager.
+
+  In particular, all ``axis_names`` become valid resource names inside the
+  managed block and can be used e.g. in the ``axis_resources`` argument of
+  :py:func:`xmap`.
```

Comment:
Cross-reference to the user-friendly introduction to these concepts?

Reply:
Done. Linked to pjit and multi-process doc.